etn-ccis/blui-react-native-themes
# Add blui-ci github workflow

Fixes BLUI-5685.



#### Changes proposed in this Pull Request:

- add blui ci github workflow
- remove circleCI config and stopped build in circleCI dashboard



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- commit small change to see jobs run and remove commit or
- see last job run here https://github.com/etn-ccis/blui-react-native-themes/actions/runs/8722657968



#### Any specific feedback you are looking for?

-
